src/hooks/useArticleLinksNavigation.js

```diff
@@ -173,7 +173,7 @@ const findVisibleLinks = (container, galleryItems) => {
 
 const isImageInGallery = (galleryItems, link) => {
   const aElement = link.querySelector('a')
-  if (!aElement) {
+  if (!aElement || !galleryItems) {
```

Comment:
never try but it is possible galleryItems is `[]` ?

Reply:
`galleryItems` is `[]` in article with no images like http://localhost:8080/#/article/fr/Veto

Reply:
@medied if I do either

``` 
const isImageInGallery = ([], link) => {
  const aElement = link.querySelector('a')
```

or 

```
const isImageInGallery = (galleryItems, link) => {
  const aElement = []
```

the navigation is broken (can't use any buttons or leave the section) when the image in the "Attack" section is loaded in http://0.0.0.0:8080/#/article/en/2010_Bratislava_shooting.

is either of the outcomes possible with ur code?

Reply:
no

the first code is invalid, the second code `aElement` will never be empty array

> querySelector returns either element or null